k8s-daemonset/README.md

```diff
@@ -149,6 +149,15 @@ And lastly, once you have linkerd deployed, you can deploy
 kubectl apply -f https://raw.githubusercontent.com/BuoyantIO/linkerd-viz/master/k8s/linkerd-viz.yml
 ```
 
+#### Daemonsets + Ingress Controller
+
+To router to external requests using ingress resources, deploy linkerd as an ingress controller, and create an ingress resource:
```

Comment:
Typo here -- I think it should be:

> To route external requests using ingress resources, ...

Also, tiny nit: mind wrapping at 80 characters?